pkgs/development/python-modules/pyqldb/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, boto3, amazon-ion, ionhash, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "pyqldb";
+  version = "3.2.2";
+
+  src = fetchFromGitHub {
+    owner = "awslabs";
+    repo = "amazon-qldb-driver-python";
+    rev = "v${version}";
+    sha256 = "sha256-TKf43+k428h8T6ye6mJrnK9D4J1xpIu0QacM7lWJF7w=";
+  };
+
+  propagatedBuildInputs = [ boto3 amazon-ion ionhash ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  pytestCheckPhase = ''
+    export AWS_DEFAULT_REGION=us-east-1
+    pytest tests/unit
+  '';
```

Comment:
```suggestion
  preCheck = ''
    export AWS_DEFAULT_REGION=us-east-1
  '';
```

Reply:
They have both unit and integration tests, I’m guessing we need to avoid the integration tests.

https://github.com/awslabs/amazon-qldb-driver-python/tree/master/tests

Also if I remember correctly, when I didn’t specify the `pytest` command it didn’t execute the tests, but will test and verify again. Not sure why

Reply:
I think we want to add

```nix
pytestCheckArgs = [
  "tests/unit"
];
```

Reply:
I see that there is no such thing yet? Should I add another PR with addition of support for `pytestCheckArgs`?

I have verified that the integration tests won't work.

Reply:
Just noticed there is `pytestFlagsArray` that fills this purpose.

Reply:
Oh, yeah use that one.